Packages.Data.props

```diff
@@ -38,8 +38,8 @@
   <!-- All csproj dependencies. -->
   <ItemGroup>
     <PackageReference Update="NUnit" Version="3.13.2" />
-    <PackageReference Update="System.ClientModel" Version="1.1.0-beta.3" />
-    <PackageReference Update="Azure.Core" Version="1.42.0" />
+    <PackageReference Update="System.ClientModel" Version="1.4.0" />
```

Comment:
do we have to upgrade this in current PR? @m-nash is upgrading it in https://github.com/Azure/autorest.csharp/pull/5297

Reply:
Yeah I was hitting downgrade warnings.

Reply:
I can wait for that to merge.